core/sdk-core/src/main/java/software/amazon/awssdk/core/interceptor/SdkExecutionAttribute.java

```diff
@@ -63,9 +64,9 @@ public class SdkExecutionAttribute {
     public static final ExecutionAttribute<String> SERVICE_NAME = new ExecutionAttribute<>("ServiceName");
 
     /**
-     * The key under which the service protocol is stored.
+     * The key under which the protocol metadata is stored.
      */
-    public static final ExecutionAttribute<String> SERVICE_PROTOCOL = new ExecutionAttribute<>("ServiceProtocol");
+    public static final ExecutionAttribute<SdkProtocolMetadata> PROTOCOL_METADATA = new ExecutionAttribute<>("ProtocolMetadata");
```

Comment:
`ExecutionAttribute<String> SERVICE_PROTOCOL` was added as part of this PR, so should be okay.
Moving to `SdkInternalExecutionAttribute` since users shouldn't be setting this